### ISSUE TITLE
Makefile.PL Line 16 is missing a semicolon

### ISSUE THREAD

**Issue by jaykao0918** [lhost/sendxmpp#15]
I think Makefile.PL is missing a semicolon at the end of line 16, making it unable to run.

**Comment by TrurlMcByte** [comment]
```diff
diff --git a/Makefile.PL b/Makefile.PL
index bdcfe22..9c54cf4 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -13,7 +13,7 @@ WriteMakefile (
        ABSTRACT => 'Script for sending XMPP messages (Jabber client)',
        PREREQ_PM => {
                'Getopt::Long'          => 2.00,
-               'Net::XMPP'                     => 1.0
+               'Net::XMPP'                     => 1.0,
                'AnyEvent::XMPP'        => 0.55,
        },
        VERSION_FROM => 'sendxmpp',
```

**Comment by lhost** [comment]
Fixed in 4114203d91323e46b8c95e2c839a49ad6f6a587c.
Please use pull request next time. Thanks.
